gubernator/templates/filtered_log.html

```diff
@@ -15,11 +15,7 @@
 		<label><input type="checkbox" name="wrap"{% if wrap %} checked{% endif %}> Line wrapping on</label>
 		% for folder, filenames in all_logs.items()|sort
 			<hr>
-			% if folder|basename == ""
-				% set artifact = folder|dirname|basename
-			% else
-				% set artifact = folder|basename
-			% endif
+			% set artifact = folder|basename or folder|dirname|basename
```

Comment:
if this is only used once, just put it inside the h4. no need for a set directive.